cmd/zoekt-sourcegraph-indexserver/queue_test.go

```diff
@@ -173,9 +194,9 @@ func TestQueue_Integration_DebugQueue(t *testing.T) {
 	actualOutput := normalizeDebugOutput(string(raw))
 
 	expectedOutput := `
-Position        Name            ID              IsOnQueue       Branches
-0               item-1          1               true            HEAD@stillQueued
-1               item-0          0               false           HEAD@popped
+Position        Name            ID              IsOnQueue       TimeSpentAwaitingIndex Branches
+0               item-1          1               true            *                      HEAD@stillQueued
```

Comment:
you could put a sample time here? Although probably clearer to just use `*`

Reply:
I think the asterisk is clearer, otherwise we might be additionally confused about where that particular sample time came from.